packages/dev/inspector-v2/src/components/properties/frameGraph/frameGraphTasksProperties.tsx

```diff
@@ -0,0 +1,19 @@
+import type { FrameGraph } from "core/index";

```

Comment:
Since this file is also just one component for the `FrameGraph` type specifically, you can collapse `frameGraphGeneralProperties.tsx` and `frameGraphTasksProperties.tsx` into one file named `frameGraphProperties.tsx`.